### PR TITLE
Evaluate query plans in query-eval

### PR DESCRIPTION
**Pull request by baitsguy**
Tested with good and bad plan

### CODE REVIEW

File: apps/query-eval/queryeval/driver.py [hunk 1]
```diff
@@ -255,12 +255,30 @@ def do_eval(self, _query: QueryEvalQuery, result: QueryEvalResult) -> QueryEvalR
...
 
+        # Evalute query plans
+        if not _query.expected_plan:
+            console.print("[yellow]:construction: No expected query plan found, skipping.. ")
+        elif not result.plan:
+            console.print("[yellow]:construction: No computed query plan found, skipping.. ")
+        else:
+            plan_diff = _query.expected_plan.compare(result.plan)
+            if len(plan_diff) == 0:
+                console.print("[green]\U00002714 Plan correct")
```
Comment (mdwelsh) [inline apps/query-eval/queryeval/driver.py:266]: You wild man

(Python supports unicode directly in string literals so you can just use that :-)

File: apps/query-eval/queryeval/driver.py [hunk 2]
```diff
@@ -255,12 +255,30 @@ def do_eval(self, _query: QueryEvalQuery, result: QueryEvalResult) -> QueryEvalR
...
+            else:
+                console.print("[red]:x: Plan incorrect")
```
Comment (mdwelsh) [inline apps/query-eval/queryeval/driver.py:268]: Maybe "Plan mismatch" (and "Plan match") since "correct" is of course dependent on many things...

**Review by mdwelsh — APPROVED** [review]

File: apps/query-eval/queryeval/driver.py (hunk 1 above)
Comment (baitsguy) [reply apps/query-eval/queryeval/driver.py:266]: Fine, what's the fun in that though

File: apps/query-eval/queryeval/driver.py (hunk 2 above)
Comment (baitsguy) [reply apps/query-eval/queryeval/driver.py:268]: Done

**Pull request merged**